sovrin_common/identity.py

```diff
@@ -10,30 +11,27 @@
 class Identity(GeneratesRequest):
```

Comment:
What do you think about `Identity` being a mixin of `DidIdentity` and `GeneratesRequest` like this `class Identity(DidIdentity, GeneratesRequest):`